### PR TITLE
Updated SDC Version for PowerFlex

### PR DESCRIPTION
**Pull request by rishabhatdell**
# Description
This PR updates the SDC version for powerflex.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1359|

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Tested Driver Install in OCP 4.15 and OCP 4.16 with SDC running successfully

### CODE REVIEW

**Review by AkshaySainiDell — COMMENTED** [review]
@rishabhatdell Any changes required in e2e ?

**Comment by rishabhatdell** [comment]
> @rishabhatdell I dont think so you need to change the SDC version in every k8s yaml file. Since we are supporting this new SDC version on csm-1.11, change the SDC version in the latest k8s version file. Changing SDC version on earlier k8s version can face issues with kernel version. Please discuss this with your team.

@abhi16394 In previous SDC update PR: https://github.com/dell/csm-operator/pull/321/files, SDC version was updated in all the k8s files, I referred to this while making the changes.

File: operatorconfig/driverconfig/common/k8s-1.24-values.yaml [hunk 1]
```diff
@@ -24,7 +24,7 @@ images:
   externalhealthmonitorcontroller: registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.11.0
 
   # "images.sdcmonitor" defines the container images used to monitor sdc container
-  sdcmonitor: dellemc/sdc:4.5.1
+  sdcmonitor: dellemc/sdc:4.5.2.1
```
Comment (adarsh-dell) [inline operatorconfig/driverconfig/common/k8s-1.24-values.yaml:27]: CSM 1.11.0 supports only k8 1.28-1.30, so I think there is no need to update the version here.
But please make sure to check that what is the under neath k8 version for OCP 4.15 and 4.16.

Thanks

**Review by suryagupta4 — APPROVED** [review]

**Review by AkshaySainiDell — APPROVED** [review]

**Review by abhi16394 — APPROVED** [review]
Thanks for the changes

**Pull request merged**